### PR TITLE
test: add testing on lib

### PR DESCRIPTION
**Pull request by mazipan**
## Description

Add unit testing

## Current Tasks

- [x] lib/Meta
- [x] lib/jsonLdGenerator

### CODE REVIEW

**Review by zainfathoni — APPROVED** [review]
🧪 💯

File: lib/__tests__/date-utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  afterAll(() => {
+    dateSpy.mockRestore();
+  });
+
+  it("should return correct long date", () => {
+    // Node.js can not return "toLocaleString" correctly using locale id
+    // It will return using default locale en-US
+    // TODO: change this after got a workaround to make this works
+    expect(getCurrentLongDate()).toBe("June 14, 2021");
```
Comment (mazipan) [inline lib/__tests__/date-utils.test.ts:20]: asem, malahan bisa di Github Actions :-D

File: lib/__tests__/date-utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { getCurrentLongDate, getCurrentMonthAndYear } from "../date-utils";
+
+describe("getCurrentLongDate", () => {
+  let dateSpy: jest.SpyInstance;
```
Comment (zainfathoni) [inline lib/__tests__/date-utils.test.ts:4]: No need to use this, Mas. I have mine implemented on #394 using Jest utilities.

**Review by zainfathoni — CHANGES_REQUESTED** [review]

**Review by zainfathoni — APPROVED** [review]

File: lib/__tests__/date-utils.test.ts (hunk 2 above)
Comment (mazipan) [reply lib/__tests__/date-utils.test.ts:4]: Better in #394, for now I already skip the test

**Pull request merged**